src/crl/mod.rs

```diff
@@ -71,16 +71,9 @@ impl<'a> RevocationOptionsBuilder<'a> {
         self
     }
 
-    /// Treat unknown revocation status permissively, acting as if the certificate were not
-    /// revoked.
-    pub fn allow_unknown_status(mut self) -> Self {
-        self.status_requirement = UnknownStatusPolicy::Allow;
-        self
-    }
-
-    /// Treat unknown revocation status strictly, considering it an error condition.
-    pub fn forbid_unknown_status(mut self) -> Self {
-        self.status_requirement = UnknownStatusPolicy::Deny;
+    /// Customize whether unknown revocation status is an error, or permitted.
+    pub fn with_status_requirement(mut self, policy: UnknownStatusPolicy) -> Self {
```

Comment:
This seems nice except that you called the method `with_status_requirement()` while the argument is called `policy` and the type is called `UnknownStatusPolicy`. I think I'd prefer to call the method (and field) `{with_,}status_policy` for consistency.